templates/mongodb.conf.erb

```diff
@@ -180,3 +180,8 @@ syslog = <%= @syslog %>
 <% if @quiet -%>
 quiet = <%= @quiet %>
 <% end -%>
+<% if @ssl -%>
```

Comment:
What would being explicit do in this case? Should it be `if @ssl and @ssl_key and @ssl_ca` ? Seems a little verbose.